block.go

```diff
@@ -416,6 +419,35 @@ func (p *parser) htmlComment(out *bytes.Buffer, data []byte, doRender bool) int
 	return 0
 }
 
+// HTML comment, lax form
+func (p *parser) htmlComment(out *bytes.Buffer, data []byte, doRender bool) int {
+	i := p.inlineHtmlComment(out, data)
+	return p.renderHtmlBlock(out, data, i, doRender)
+}
+
+// HTML CDATA section
+func (p *parser) htmlCDATA(out *bytes.Buffer, data []byte, doRender bool) int {
+	cdataTag := "<![cdata["
+	cdataTagLen := len(cdataTag)
```

Comment:
Ahh, I keep forgetting Go is not JS or Python :-)
Of course, `const` makes more sense, I'll change that.